qcfractal/storage_sockets/models.py

```diff
@@ -53,8 +53,10 @@ def __str__(self):
     meta = {
         'collection': 'molecules',
         'indexes': [
-            {'fields': ('molecule_hash', 'molecular_formula'),
-             'unique': False
+            {'fields': ('molecule_hash',), 'unique': True},
```

Comment:
This should *likely* be unique, but is not guaranteed. We use it effectively as a very fast lookup for molecules.

Reply:
In normal cases, can we assume that molecule_formula is almost unique like molecule_hash?
If not, by which factor?

Reply:
`molecular_formula` might relate to 1-1e6 molecules, average maybe 1e2-3.

Reply:
If we will have lots of duplicates in molecule_formula then we will need another index just for molecule_formula.
Another question, I don't see that we do a combined query on ('molecule_hash', 'molecular_formula'), we always do it on either one. If this is the case, then we should have single index of each field not a combined one.

Reply:
Right, that one is pretty much an either or combination.